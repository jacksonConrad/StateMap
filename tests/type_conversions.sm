// tests built in functions

void DFA main() {
	start {
		int i = 20;
		string s = itos(i);
		i = stoi(s);

		print(s); // "20"
		print(i); // 20

		float f = 24.8;
		s = ftos(f);
		f = stof(s);

		print(s); // "24.8"
		print(f); // 24.8

		return;
	}
}