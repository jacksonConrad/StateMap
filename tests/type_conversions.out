20
20
24.8
24.8
